### PR TITLE
chore: balance = capacity of live cells + sent cells

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

**Comment by classicalliu** [comment]
> Can we add a test for this?

Tests added.

**Comment by Keith-CY** [comment]
CI failed

File: packages/neuron-wallet/tests/database/address/balance.test.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  }
+
+  it('balance = live + sent - pending', async () => {
+    const address = generateAddress('1000', '100', '300')
+    const addrs: AddressEntity[] = await AddressDao.create([address])
+    const addr = addrs[0]
+    expect(addr.balance()).toEqual((1000 + 100).toString())
+  })
+
+  it('balance is right in interface', async () => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/tests/database/address/balance.test.ts]:
```suggestion
  it('balance returned by the interface method', async () => {
```

File: packages/neuron-wallet/tests/database/address/balance.test.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    ]
+    const addrs: AddressEntity[] = await AddressDao.create(addresses)
+    const balance: bigint = addrs
+      .map(addr => BigInt(addr.balance()))
+      .reduce((result, c) => result + c, BigInt(0))
+
+    expect(balance).toEqual(BigInt(1000))
+  })
+
+  it('sent to others with 10shannon fee', async () => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/tests/database/address/balance.test.ts]:
```suggestion
  it('sent to others with 10 shannon fee', async () => {
```

File: packages/neuron-wallet/tests/database/address/balance.test.ts (hunk 1 above)
Comment (classicalliu) [reply packages/neuron-wallet/tests/database/address/balance.test.ts]: This test this focus on `balance` is right or not, not returned or not

Comment (Keith-CY) [reply packages/neuron-wallet/tests/database/address/balance.test.ts]: > This test this focus on `balance` is right or not, not returned or not

The description is a bit confusing, `balance is right in interface` means `the balance field is listed in the interface exactly`, try to use a better description, such as `the balance returned by the interface method is correct`

**Pull request merged**